src/errorDictionary.js

```diff
@@ -0,0 +1,63 @@
+import fs from "fs";
+
+let dictionary;
+
+module.exports = {
+  init: function (config) {
+    if(!dictionary && config){
+
+      // check if file
+      if(typeof(config) === 'string' && fs.existsSync(config)){
```

Comment:
what do we do if typeof(config) isn't string, or if the dictionary content isn't in JSON format, do we alert user?